prbt_support/package.xml

```diff
@@ -28,6 +28,7 @@
   <exec_depend>xacro</exec_depend>
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>pilz_control</exec_depend> <!--contains the PilzJointTrajectoryController-->
+  <exec_depend>pilz_status_indicator_rqt</exec_depend>
```

Comment:
Is the `pilz_status_indicator_rqt` already connected to the system? Or is this a node without further functionality yet? If so, leave it out, until it works correctly. Please, fix this in PR #325 too, if needed.

Reply:
It is partially functional, already. In my tests, it displayed the operation mode correctly. I want to leave it in here because it is an essential part of this whole user story.

Reply:
Ok.